examples/react-presence/.env.production

```
@@ -0,0 +1,2 @@
+VITE_YORKIE_API_ADDR='https://api.yorkie.dev'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)